docs/user/tutorials/01-00-x-forwarded-for-header.md

```diff
@@ -31,13 +28,34 @@ Run the following command:
   ```
 
 #### **Kyma Dashboard**
-1. Navigate to `Cluster Details`.
-2. Open the Istio module's configuration.
-3. Click **Edit**.
-4. Add the number of trusted proxies and click **Update**.
-   ![Add the numTrustedProxies](./assets/01-00-num-trusted-proxies-ui.svg)
+1. Navigate to Cluster Details and select Modify Modules.
+2. Choose the Istio module and select Edit.
+3. In the General section, add the number of trusted proxies.
+4. Select Save.
 <!-- tabs:end -->
 
+### Configure the externalTrafficPolicy in the Istio Custom Resource (GCP and Azure only)
+
+To make XFF header contain client's IP on GCP and Azure, it's required to set gatewayExternalTrafficPolicy to `Local`
```

Comment:
```suggestion
You can configure external traffic policy only in GCP and Azure clusters. To include the client's IP in the XFF header, set **gatewayExternalTrafficPolicy** to `Local`.
```

Reply:
We can configure it on every cloud provider. But to have XFF header forwarding the client's IP on Azure and GCP you have to set gatewayExternalTrafficPolicy to Local. On AWS it does not have any effect regarding the XFF header content.

Reply:
ahh, okay 
sorry, I didn't get this

Reply:
I've edited the suggestion